src/themes/shared/components/Input/input-styles.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ComponentSlotStylesPrepared,
+  TeamsThemeStylesProps,
+} from "@fluentui/react-northstar";
+import { Surface } from "../../../../types/types";
+import { TeamsTheme } from "../../../constants";
+
+export const inputStyles: ComponentSlotStylesPrepared<
+  NonNullable<Partial<TeamsThemeStylesProps["Input"]>>,
+  any
+> = {
+  input: (componentStyleParameters) => {
+    const {
+      variables,
+      theme: {
+        siteVariables: { colorScheme, theme },
+      },
+    } = componentStyleParameters;
+    return {
+      backgroundColor: (() => {
+        switch (variables.surface) {
+          case Surface.base:
+            return colorScheme.default.background;
+          case Surface.raised:
```

Comment:
Just double checking that this fallthrough is intentional.

Reply:
It is! Thanks for checking :)